client/src/views/users/UserContextMenu.vue

```diff
@@ -54,16 +54,30 @@ export enum UserAction {
 </script>
 
 <script setup lang="ts">
-import { IonContent, IonItem, IonLabel, IonList, popoverController, IonIcon, IonItemGroup } from '@ionic/vue';
-import { personRemove, informationCircle } from 'ionicons/icons';
+import UserDetailsModal from '@/views/users/UserDetailsModal.vue';
+import { IonContent, IonIcon, IonItem, IonItemGroup, IonLabel, IonList, modalController, popoverController } from '@ionic/vue';
+import { informationCircle, personRemove } from 'ionicons/icons';
+import { UserInfo } from '@/parsec';
 
-defineProps<{
-  isRevoked: boolean;
+const props = defineProps<{
+  user: UserInfo;
 }>();
 
 function onClick(action: UserAction): Promise<boolean> {
   return popoverController.dismiss({ action: action });
 }
+
+async function presentDetailsModal(): Promise<void> {
+  const modal = await modalController.create({
+    component: UserDetailsModal,
+    cssClass: 'user-details-modal',
+    componentProps: {
+      user: props.user,
+    },
+  });
+  await modal.present();
+  await modal.onWillDismiss();
+}
```

Comment:
maybe a popoverController.dismiss is missing here since you did not use onClick with action.
@Max-7 a thought on this?

Reply:
Yeah, always add a `await modal.dismiss()` after the `onWillDismiss()`.